apps/api/src/routes/v1_identities_createIdentity.error.test.ts

```diff
@@ -62,13 +62,6 @@ describe("when identity exists already", () => {
       },
     });
 
-    expect(res.status).toEqual(412);
-    expect(res.body).toMatchObject({
-      error: {
-        code: "PRECONDITION_FAILED",
-        docs: "https://unkey.dev/docs/api-reference/errors/code/PRECONDITION_FAILED",
-        message: "Duplicate identity",
-      },
-    });
+    expect(res.status, `expected 409, received: ${JSON.stringify(res, null, 2)}`).toEqual(409);
```

Comment:
why is the body check gone?

Reply:
Since the 409 status code was only used for both endpoints in a single place, I thought their uniqueness was enough to check the status code. 

I've added both checks back.